src/components/common/Header.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import { Menu } from 'antd';
-import { useOktaAuth } from '@okta/okta-react';
 
 export default function MainHeader() {
-  const { authService, authState } = useOktaAuth();
```

Comment:
Nice job removing the Okta hook, but it looks like authState will still be used throughout the app. I wonder if it would have been helpful to leave this line commented out with just the okta portion removed and a note stating to update the hook?

Reply:
That's a good idea, I can try that.